demos/Demo.ts

```diff
@@ -325,11 +328,34 @@ class Demo {
         }
     }
 
-    public _createExportRequest(accountId: string): SimpleRequest {
-        return {
-            appName: 'Accounts Demos',
-            accountId,
-        } as SimpleRequest;
+    public async exportFile(accountId: string) {
+        try {
+            const result = await this._accountsClient.export({
+                appName: 'Accounts Demos',
+                accountId,
+                fileOnly: true,
+            });
+            console.log('Keyguard result', result);
+            document.querySelector('#result').textContent = 'Export sucessful';
```

Comment:
The message should be conditional to the actual result flag. Because it can be `false`.